lua/gatekeeper_config.lua

```diff
@@ -10,7 +10,8 @@ function gatekeeper_init()
 	-- When gatekeeper_server is true,
 	-- Gatekeeper will run as a Gatekeeper server.
 	-- Otherwise, it will run as a grantor server.
-	local gatekeeper_server = true
+	--local gatekeeper_server = true
+	local gatekeeper_server = false
```

Comment:
Let's remove this update to avoid flipping between Gatekeeper and Grantor servers in between merge of pull requests.